pkg/cmd/common.go

```diff
@@ -104,3 +118,15 @@ func (co *CostOptions) Validate() error {
 
 	return nil
 }
+
+// Binds the flag with viper environment variable and ensures the order of precendence
+// command line > environment variable > default value
+func bindAFlagToViperEnv(cmd *cobra.Command, v *viper.Viper, flag string) {
```

Comment:
Does viper have a different precedence if we don't use a helper function like this?

Reply:
Yes it would be 
use-proxy value > Default > environment variable if there is no check. 

Test case conducted were: 

 Test case 1: `go run cmd/kubectl-cost/kubectl-cost.go namespace` --show-all-resources -> doesn’t use proxy -> Expected as default is false

Test case 2: `KUBECTL_COST_USE_PROXY=true go run cmd/kubectl-cost/kubectl-cost.go namespace --show-all-resources` -> should use environment variable as it’s overridden by the KUBECTL_COST_USE_PROXY


Test case 3: `KUBECTL_COST_USE_PROXY=true  go run cmd/kubectl-cost/kubectl-cost.go namespace --show-all-resources  —use-proxy=false` -> doesnt use proxy as environment value is overridden by --use-proxy value